news/2022-09-16-project-loved-september-2022.md

```diff
@@ -0,0 +1,341 @@
+---
+layout: post
+title: "Project Loved: September 2022"
+date: 2022-09-16 08:00:00 +0000
+---
+
+The captains at Project Loved present to you a carefully crafted curation of classic creations — or in other words, the next round is here! Come check out the maps and cast your votes.
+
+![](/wiki/shared/news/banners/project-loved-2.jpg)
+
+For those unfamiliar, Project Loved hosts monthly votings to decide which nominated maps get added to the [Loved category](https://osu.ppy.sh/beatmapsets?s=loved). You can click on each map's banner in this post to go to its poll on the forums, or visit [the forum](https://osu.ppy.sh/community/forums/120) for a list of all of the polls. Polls will be open for 10 days, so make sure to cast your votes in time! More info can be found on the [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh)!
+
+Detailed results for the previous round can be found on the [poll results page of the website](https://loved.sh/statistics) or in the following forum posts:
+
+- [osu!](https://osu.ppy.sh/community/forums/posts/8768240)
+- [osu!taiko](https://osu.ppy.sh/community/forums/posts/8768239)
+- [osu!catch](https://osu.ppy.sh/community/forums/posts/8768238)
+- [osu!mania](https://osu.ppy.sh/community/forums/posts/8768237)
+
+The main update this time around is that the osu! captains have closed their application and decided on the next captains that'll be helping out. In the coming months, you'll get to know these fine gentlemen as they share maps here and open Loved polls:
```

Comment:
doesn't quite make sense otherwise

```suggestion
The main update this time around is that the osu! captains have closed their application form and decided on the next captains that'll be helping out. In the coming months, you'll get to know these fine gentlemen as they share maps here and open Loved polls:
```